tools/packaging/generate_debian_package.sh

```diff
@@ -12,11 +12,23 @@
 # Please note that is must run from main branch.
 # Best running this from inside a container.
 # The result are writing to the location from which you would execute the script (not where the script is located).
-# Version number is the tag number. Currently this would only generate a package for amd64.
+# Version number is the tag number.
+# Params:
+#	* arch type: either amd64 or arm64 (aarch64)
+#	* optional location to the binary to place at the package
 
 
 set -eu
 
+ARCH_VAL=amd64
+if [ $# -eq 1 ]; then
+	if [ "$1" != "${ARCH_VAL}" -a "$1" != "aarch64" ]; then
```

Comment:
you do not need to pass it by argument. you can do it with `uname -m`
See here for example: 
https://github.com/romange/image-bakery/blob/d8785cb97b20950ca799f06df63a9042f5bf4c79/provision/cloudconfig.cue#L81

Reply:
OK, I will just make sure that this is working (actually we just need to make sure that this not x86 and set it to aarch64)